staging/src/github.com/kubeedge/mapper-framework/_template/mapper/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM golang:1.17-alpine AS builder
+FROM golang:1.20-alpine AS builder
```

```suggestion
FROM golang:1.20.10-alpine3.18
```
Seems we used the above version in all other dockerfiles cc @Shelley-BaoYue